package.json

```diff
@@ -8,6 +8,7 @@
     "create-nos-dapp": "lib/index.js"
   },
   "scripts": {
+    "test": "echo \"No tests yet\"",
```

Comment:
Should we make this intentionally fail until tests are added?  I think it will help to signify they're not there until we're ready to add them.  Plus it would better match the api-functions repo.  i.e.:

```js
"test": "echo \"Error: no test specified\" && exit 1"
```

Reply:
Yeah, that's the only drawback.  Admins can still merge, and I'm fine with that approach as long as we're planning to add a test suite in the next PR.  If not, it might be better just to set up an empty test suite as part of this PR.